### PR TITLE
Update lagscope build steps

### PR DESCRIPTION
**Pull request by LiliDeng**
Change build lagscope steps,  which bring in by https://github.com/microsoft/lagscope/pull/18

### CODE REVIEW

File: Testscripts/Linux/utils.sh [hunk 1]
```diff
@@ -2622,21 +2624,21 @@ function install_lagscope () {
 	case "$DISTRO_NAME" in
 		oracle|rhel|centos)
 			install_epel
-			yum -y --nogpgcheck install libaio sysstat git bc make gcc wget
```
Comment (santoshx) [inline Testscripts/Linux/utils.sh:2625]: You'll still need the 'make' because cmake will internally use make.

Comment (LiliDeng) [reply Testscripts/Linux/utils.sh:2625]: I tested pass, probably it has make by default, anyway I will add it back, thanks for review.

**Review by santoshx — APPROVED** [review]

**Comment by LiliDeng** [comment]
Test results -

```
[LISAv2 Test Results Summary]
Test Run On           : 07/16/2019 18:47:54
ARM Image Under Test  : OpenLogic : CentOS : 7.6 : latest
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:14

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 NETWORK              PERF-NETWORK-TCP-LATENCY-Synthetic                                                PASS                11.88 
	Minimum Latency : 161.886 
	Maximum Latency : 16156.197 
	Average Latency : 407.634 

[LISAv2 Test Results Summary]
Test Run On           : 07/16/2019 18:47:32
ARM Image Under Test  : credativ : Debian : 9-backports : Latest
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:19

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 NETWORK              PERF-NETWORK-TCP-LATENCY-Synthetic                                                PASS                16.23 
	Minimum Latency : 179.052 
	Maximum Latency : 20901.918 
	Average Latency : 662.513 
```

**Comment by santoshx** [comment]
I posted a PR to lagscope that removes dependency on C++.
https://github.com/microsoft/lagscope/pull/20

Please update this PR accordingly.

**Review by santoshx — APPROVED** [review]

**Pull request merged**